hax/hax/hax.py

```diff
@@ -28,6 +27,7 @@
 from hax.motr import Motr
 from hax.motr.delivery import DeliveryHerald
 from hax.motr.ffi import HaxFFI
+from hax.motr.planner import WorkPlanner
```

Comment:
In design document this thing is called WorkPlanner, I didn't come up to a better name :-)

Scheduler sounds to me quite the same to planner (scheduler is more like connected with a calendar - we don't have it, planner is just about making a plan). This thing is a synchronization point that a kind of "plans" which work can be done right now and which cannot. And when a piece of work becomes eligible for processing, it will wake up the blocked thread.

Reply:
Okay, fair enough.